mappings/net/minecraft/block/entity/SignBlockEntity.mapping

```diff
@@ -36,5 +38,8 @@ CLASS net/minecraft/class_2625 net/minecraft/block/entity/SignBlockEntity
 		ARG 3 filteredText
 	METHOD method_33828 parseTextFromJson (Ljava/lang/String;)Lnet/minecraft/class_2561;
 		ARG 1 json
+ 	METHOD method_33829 updateSign (ZLjava/util/function/Function;)[Lnet/minecraft/class_5481;
```

Comment:
```suggestion
	METHOD method_33829 updateSign (ZLjava/util/function/Function;)[Lnet/minecraft/class_5481;
```
Build is still failing because of this. (Sorry for overlooking it on my first review. This is why it's worth checking build locally before push.)